app/controllers/ops_controller/settings/label_tag_mapping.rb

```diff
@@ -153,7 +153,7 @@ def label_tag_mapping_add(entity, label_name, cat_description)
 
     # UI currently can't allow 2 mappings for same (entity, label).
     if Classification.find_by_name(cat_name)
-      add_flash(_("Mapping for %{entity}, %{label} already exists") %
+      add_flash(_("Mapping for %{entity}, Label %{label} already exists") %
```

Comment:
@h-kataria I added double quotes to be consistent with 
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/ops_controller/settings/label_tag_mapping.rb#L175 
and
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/ops_controller/settings/label_tag_mapping.rb#L193